Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -3014,7 +3016,7 @@ def gen2021HiMix(fragment,howMuch):
 steps['ALCAEXPHI']=merge([{'-s':'ALCA:PromptCalibProd+PromptCalibProdSiStrip+PromptCalibProdSiStripGains+PromptCalibProdSiStripGainsAAG',
                   '--scenario':'HeavyIons'},steps['ALCAEXP']])
 steps['ALCAEXPTE']={'-s':'ALCA:PromptCalibProdEcalPedestals',
-                    '--conditions':'auto:run2_data_relval',
+                    '--conditions':'auto:run2_data',
```

Comment:
@cms-sw/alca-l2

I need feedback from AlCa experts to finalise this PR.

I made this change assuming that the difference between `auto:run2_data` and `auto:run2_data_relval` was only the L1T-Menu tag, but that assumption was wrong, as a few other tags are different
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/124X_dataRun2_v2/124X_dataRun2_relval_v2

The different `EcalPedestals*` tags might explain the differences in DQM comparisons of wf `1010.0` in the [latest comparisons of this PR](https://github.com/cms-sw/cmssw/pull/40481#issuecomment-1382205870).

Questions:
 - is it wrong to assume that `auto:X_relval` differs from `auto:X` only in the L1T-Menu tag(s) ?
 - what is the appropriate GT in this particular case (step 3 of wf `1010.0`) ?

Reply:
hi @missirol 
your assumption was correct, it seems to be a mistake from our predecessors. We will fix it. We are actually thinking about it to be a symbolic GT where it takes the `auto:X_relval` and adds the fix L1T-menu from the code.

`ALCAEXPTE` step is supposed to mimic express and do a PCL test. Thus it's the Express GT that should be consumed. Given that that key doesnt exists for Run-2, I'd go with `auto:run2_data`

Reply:
Okay. Thanks for the feedback!